src/lib/linux-api/src/bindings.rs

```diff
@@ -2222,6 +2488,803 @@ fn bindgen_test_layout_rseq() {
+pub const LINUX_NDUSEROPT_SRCADDR: linux__bindgen_ty_66 = 1;
+pub const LINUX___NDUSEROPT_MAX: linux__bindgen_ty_66 = 2;
+pub type linux__bindgen_ty_66 = ::core::ffi::c_uint;
+pub const LINUX_TCA_ROOT_UNSPEC: linux__bindgen_ty_67 = 0;
+pub const LINUX_TCA_ROOT_TAB: linux__bindgen_ty_67 = 1;
+pub const LINUX_TCA_ROOT_FLAGS: linux__bindgen_ty_67 = 2;
+pub const LINUX_TCA_ROOT_COUNT: linux__bindgen_ty_67 = 3;
+pub const LINUX_TCA_ROOT_TIME_DELTA: linux__bindgen_ty_67 = 4;
+pub const LINUX___TCA_ROOT_MAX: linux__bindgen_ty_67 = 5;
+pub type linux__bindgen_ty_67 = ::core::ffi::c_uint;
 #[doc = " struct linux_clone_args - arguments for the clone3 syscall\n @flags:        Flags for the new process as listed above.\n                All flags are valid except for LINUX_CSIGNAL and\n                LINUX_CLONE_DETACHED.\n @pidfd:        If LINUX_CLONE_PIDFD is set, a pidfd will be\n                returned in this argument.\n @child_tid:    If LINUX_CLONE_CHILD_SETTID is set, the TID of the\n                child process will be returned in the child's\n                memory.\n @parent_tid:   If LINUX_CLONE_PARENT_SETTID is set, the TID of\n                the child process will be returned in the\n                parent's memory.\n @exit_signal:  The exit_signal the parent process will be\n                sent when the child exits.\n @stack:        Specify the location of the stack for the\n                child process.\n                Note, @stack is expected to point to the\n                lowest address. The stack direction will be\n                determined by the kernel and set up\n                appropriately based on @stack_size.\n @stack_size:   The size of the stack for the child process.\n @tls:          If LINUX_CLONE_SETTLS is set, the tls descriptor\n                is set to tls.\n @set_tid:      Pointer to an array of type *pid_t. The size\n                of the array is defined using @set_tid_size.\n                This array is used to select PIDs/TIDs for\n                newly created processes. The first element in\n                this defines the PID in the most nested PID\n                namespace. Each additional element in the array\n                defines the PID in the parent PID namespace of\n                the original PID namespace. If the array has\n                less entries than the number of currently\n                nested PID namespaces only the PIDs in the\n                corresponding namespaces are set.\n @set_tid_size: This defines the size of the array referenced\n                in @set_tid. This cannot be larger than the\n                kernel's limit of nested PID namespaces.\n @cgroup:       If LINUX_CLONE_INTO_CGROUP is specified set this to\n                a file descriptor for the cgroup.\n\n The structure is versioned by size and thus extensible.\n New struct members must go at the end of the struct and\n must be properly 64bit aligned."]
```

Comment:
If you mean the `#[doc = ...]` line, this is created by bindgen so I think it's okay. It takes the code comments from the linux header files and adds the text as rustdoc comments for the rust structure. I'm not sure why they use `#[doc = "a really long line"]` rather than using a multi-line `///` comment, but I'm guessing it's to handle whitespace formatting more consistently.